packages/devextreme/js/ui/text_box/ui.text_editor.base.js

```diff
@@ -924,6 +924,10 @@ const TextEditorBase = Editor.inherit({
         } else {
             this.callBase();
         }
+
+        this.option('validationStatus', 'valid');
+        this._showValidMark = false;
+        this._renderValidationState();
```

Comment:
do we really need to rerender all the validation state? Cannot we just move the code related to validMark to a separate method and call it here?

Reply:
sure